### DISCUSSION TITLE
Docker question

### DISCUSSION THREAD

**Discussion by petrosmm** [directus/directus#9950] [Archive]
I don't pretend to be an expert but I was curious to ask: how come I can use `/bin/sh`, but `/bin/bash` won't work? I can use bash with all my other containers in Docker using the portainer interface.

**Comment by rijkvanzanten** [comment]
`sh` is a different shell from `bash`. In order to reduce the total size of the image, certain images omit the full bash if it isn't used for anything 🙂

**Comment by azrikahar** [comment]
Just to clarify, `bash` won't work for alpine based image (by default, as it's omitted), which is what Directus is based off of. When you say you can use bash with all other containers, I believe it's a case of all the other containers just so happens to _not_ be based on alpine. You will face the same issue with `nginx:alpine` docker image for example, but not `nginx`  👍
